javascript/modules/client/README.md

```diff
@@ -1,105 +1,115 @@
-# Aragon JS SDK (client)
+# @aragon/sdk-client
 
-<!--
-Congrats! You just saved yourself hours of work by bootstrapping this project with TSDX. Let’s get you oriented with what’s here and how to use it.
+@aragon/sdk-client contains the high level operations for interacting with the Aragon ecosystem
 
-> This TSDX setup is meant for developing libraries (not apps!) that can be published to NPM. If you’re looking to build a Node app, you could use `ts-node-dev`, plain `ts-node`, or simple `tsc`.
+## Installation
 
-> If you’re new to TypeScript, checkout [this handy cheatsheet](https://devhints.io/typescript)
-
-## Commands
-
-TSDX scaffolds your new library inside `/src`.
-
-To run TSDX, use:
+Use [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) to install @aragon/sdk-signing.
 
 ```bash
-npm start # or yarn start
+npm install @aragon/sdk-client
+yarn add @aragon/sdk-client
 ```
 
-This builds to `/dist` and runs the project in watch mode so any edits you save inside `src` causes a rebuild to `/dist`.
-
-To do a one-off build, use `npm run build` or `yarn build`.
-
-To run tests, use `npm test` or `yarn test`.
-
-## Configuration
-
-Code quality is set up for you with `prettier`, `husky`, and `lint-staged`. Adjust the respective fields in `package.json` accordingly.
-
-### Jest
-
-Jest tests are set up to run with `npm test` or `yarn test`.
-
-### Bundle Analysis
-
-[`size-limit`](https://github.com/ai/size-limit) is set up to calculate the real cost of your library with `npm run size` and visualize the bundle with `npm run analyze`.
-
-#### Setup Files
-
-This is the folder structure we set up for you:
-
-```txt
-/src
-  index.tsx       # EDIT THIS
-/test
-  blah.test.tsx   # EDIT THIS
-.gitignore
-package.json
-README.md         # EDIT THIS
-tsconfig.json
+## Usage
+
+#### DAO Creation
+
+###### DAO ERC20 Voting
+
+```ts
+// For local testing
+const contextParams: ContextParams = {
+    network: 31337,
+    signer: new Wallet("privateKey"),
+    daoFactoryAddress: "daoFactoryAddress",
+    web3Providers: ["http://localhost:8545"],
+};
+const context = new Context(contextParams);
+
+const client = new ClientDaoERC20Voting(context);
+
+const daoCreationParams: ICreateDaoERC20Voting = {
+    daoConfig: {
+        name: "ERC20VotingDAO_" + Math.floor(Math.random() * 9999) + 1,
+        metadata: "0x1234",
+    },
+    tokenConfig: {
+        addr: "0x0000000000000000000000000000000000000000",
+        name:
+            "TestToken" +
+            (Math.random() + 1)
+                .toString(36)
+                .substring(4)
+                .toUpperCase(),
+        symbol:
+            "TEST" +
+            (Math.random() + 1)
+                .toString(36)
+                .substring(4)
+                .toUpperCase(),
+    },
+    mintConfig: {
+        receivers: [
```

Comment:
Why not having  `[ {address: "...", balance: BigInt() }, ... ]`?
If the contract needs another format, the SDK can zip and handle the data